rust/processor/src/db/postgres/schema.rs

```diff
@@ -423,6 +423,35 @@ diesel::table! {
 
 diesel::table! {
     current_fungible_asset_balances (storage_id) {
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 66]
+        asset_type_v2 -> Nullable<Varchar>,
+        #[max_length = 1000]
+        asset_type_v1 -> Nullable<Varchar>,
+        is_primary -> Nullable<Bool>,
```

Comment:
This shouldn't be nullable to match with the legacy schema, but I see some columns that are null is db

Reply:
@bowenyang007

Reply:
Synced offline, this will be fixed in next PR

Reply:
this isn't related to this PR. only popped up due to renaming table. We'll address it next PR.